app/src/ep.config.json

```diff
@@ -1,8 +1,7 @@
 {
   "cortexApi": {
-    "path": "/cortex",
-    "scope": "vestri",
-    "pathForProxy": "http://localhost:9080"
+    "path": "https://vestri-us.epdemos.com/cortex",
```

Comment:
Please don't override this configuration

Reply:
I have reverted it back, I added my config file by accident